Microsoft.Toolkit.Uwp/Helpers/BackgroundTaskHelper.cs

```diff
@@ -43,6 +43,12 @@ public static bool IsBackgroundTaskRegistered(Type backgroundTaskType)
         /// <returns>Background Task that was registered with the system</returns>
         public static async Task<BackgroundTaskRegistration> RegisterAsync(Type backgroundTaskType, IBackgroundTrigger trigger, bool enforceConditions = true, params IBackgroundCondition[] conditions)
         {
+            // Check if the task is already registered.
```

Comment:
Is it possible to have the same background task registered twice, but respond to different trigger? It never crossed my mind.